libs/admin/src/lib/state/admin.actions.ts

```diff
@@ -0,0 +1,74 @@
+import { createAction, props } from '@ngrx/store';
+import { User, Workgroup } from '../services/admin.service';
+
+export const findUser = createAction(
+  '[Asset Search] Find user',
```

Comment:
Replace the prefix on all of these actions - we are not in the asset search state here.
```suggestion
  '[Admin] Find user',
```